zipline/finance/performance/tracker.py

```diff
@@ -385,11 +400,15 @@ def handle_market_close(self, dt, data_portal):
 
             benchmark_value = self.all_benchmark_returns[completed_session]
 
+            portfolio = self.get_portfolio(False)
```

Comment:
Style note: it's almost always clearer to pass boolean function arguments by keyword.  `self.get_portfolio(False)` provides very little information to a reader, but `self.get_portfolio(performance_needs_update=False)` is much more useful.